### ISSUE TITLE
Better differentiate icons in admin user module (TYPO3 version 7)

### ISSUE THREAD

**Issue by erredeco** [TYPO3/TYPO3.Icons#223]
as Xavier Perseguers noticed, color-blind people could find hard to understand which user is an administrator and which is not, because the icons superimposed on the photo differs only by the "shirt" color. See attached screenshot.
![color-blind](https://cloud.githubusercontent.com/assets/3647646/10715337/ce75607c-7b13-11e5-8852-f368ffd1e23e.png)

Suggestion

> Suggestion for admin icon, "inspired" by some sites and an IRC client: put a crown on the head or use only a crown. (Just make sure that colour is not the only difference between two icons)

**Comment by rfoucard** [comment]
Hi,
open for discussion: what about a silhouette inside a bigger shape: 

- a simple circle for a basic user
- a shield for an admin

![Image](https://github.com/user-attachments/assets/092b0adb-dead-4048-bf9f-fce50fbe2a31)

**Comment by erredeco** [comment]
I agree with the surrounding shape approach because the icon tends to be very small and adding a tiny detail (like a cog or a key) could become virtually invisible.

Other options could be (for admin user):
- gear (cog)
- hexagon

**Comment by benjaminkott** [comment]
All considerations need to include that Icons have a max working area of 16x16px (14x14px effective), the 1px gap is only used for small overhangs. All circles have a size of 14x14px, the same goes for shields.

All icons must be crisp on non-retina devices, displayed at 16x16px
All icon shapes should match existing look and feel.

See: 
- https://typo3.github.io/TYPO3.Icons/icons/actions/actions-check-circle.html
- https://typo3.github.io/TYPO3.Icons/icons/actions/actions-shield-typo3.html


![Image](https://github.com/user-attachments/assets/beba74b2-fabe-485c-b3e8-64e8bfee873b)

**Comment by erredeco** [comment]
How about an approach like this one? (test0 is non-admin) - just a wild suggestion

![Image](https://github.com/user-attachments/assets/6ad0c056-20b8-4610-ab4c-37f452dbefd3)


![Image](https://github.com/user-attachments/assets/d50a66c7-119b-4e5b-a51b-ed61e0b86df4)

**Comment by NeoBlack** [comment]
The actual problem isn't the color, because even colorblind people don't know why an admin wears blue or orange ;)

When I look at other systems with similar roles, a user seems to be represented as a person, and an admin as a person with the addition of a gear. Alternatively, a person with the addition of a shield, but I associate that more with certification or security.

Then we have the front-end users, that - of course - wear green ;)

Specifically, I would go with a person and a person + gear. I don't know if that's feasible at 16x16px or 14x14px.

We could also use the same icon and mark admins (or other roles) with an icon overlay. This is just an idea.

Overall, however, we also have two views: The BE-User-Module and the List-Module. We should streamline these views and find icons that work in all contexts.

**List Modul, BE Users:**
![Image](https://github.com/user-attachments/assets/8230439a-4deb-413b-b9a0-3889b5195dc1)

**List Module, FE Users:**
![Image](https://github.com/user-attachments/assets/48ada901-4b4f-4b7c-87fb-bef4f8f81b98)

**BE User Module**
![Image](https://github.com/user-attachments/assets/7ec66452-b686-4bff-ae1b-53d5c520ff50)

**Comment by rfoucard** [comment]
Something more like this?

![Image](https://github.com/user-attachments/assets/af1a250b-ba1b-4525-8ee7-361ed72d9307)

in that case I would recommand to only put a gear on the bottom right for the BE User Module for the admin users then.
